clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v1/deploy/KubernetesUtil.groovy

```diff
@@ -34,6 +35,8 @@ class KubernetesUtil {
   static String DEPRECATED_SERVER_GROUP_KIND = "ReplicationController"
   static String SERVER_GROUP_KIND = "ReplicaSet"
   static String DEPLOYMENT_KIND = "Deployment"
+  static String SERVER_GROUP_API_VERSION = new ReplicaSet().getApiVersion()
+  static String DEPLOYMENT_API_VERSION = new Deployment().getApiVersion()
```

Comment:
When we bump the underlying client version, do we now risk breaking user's deployments that depended on a certain workload version?

Reply:
Thank you for review, I change the way for api version detection.

Reply:
How about this way?